### PR TITLE
minor tweaks

### PR DESCRIPTION
**Pull request by badersur**

### CODE REVIEW

**Comment by badersur** [comment]
Welcome. These changes will need some time to be updated and I'll add **ASP.NET** in front of MVC, and may change other related files!

**Comment by scottaddie** [comment]
@badersur Those sound like good changes. Let me know when it's ready for review again.

**Comment by badersur** [comment]
@scottaddie Could you take a look at my changes?

File: aspnet/mvc/overview/getting-started/recommended-resources-for-mvc.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ by [Rick Anderson](https://github.com/Rick-Anderson)
 - [Selecting Cultures for Localization in ASP.NET](http://weblog.west-wind.com/posts/2014/Mar/27/Auto-Selecting-Cultures-for-Localization-in-ASPNET)
 - [Nesting Layouts in MVC](http://astutelogic.com/nesting-layouts-in-mvc/) by Russell Waite
 - [Create an ASP.NET MVC app with auth and SQL DB and deploy to Azure App Service](https://www.windowsazure.com/en-us/develop/net/tutorials/web-site-with-sql-database/)
```
Comment (badersur) [inline aspnet/mvc/overview/getting-started/recommended-resources-for-mvc.md]: @scottaddie This tutorial points to ASP.NET Core page and maybe it should be removed (seems to be redundant)

Comment (scottaddie) [reply aspnet/mvc/overview/getting-started/recommended-resources-for-mvc.md]: Please remove this one.

Comment (badersur) [reply aspnet/mvc/overview/getting-started/recommended-resources-for-mvc.md]: Removed :)

**Comment by scottaddie** [comment]
@badersur Thank you for cleaning this up for us!

**Pull request merged**